pkgs/by-name/sn/snapcraft/package.nix

```diff
@@ -5,26 +5,49 @@
   lib,
   makeWrapper,
   nix-update-script,
-  python3Packages,
+  python3,
   squashfsTools,
   stdenv,
 }:
-python3Packages.buildPythonApplication rec {
+
+let
+  python = python3.override {
+    packageOverrides = self: super: {
+      pydantic = self.pydantic_1;
```

Comment:
This line can even be removed once https://github.com/NixOS/nixpkgs/pull/329679 is merged.

Reply:
Snapcraft still uses `pydantic_1` in its `package.nix`. This is just about dependencies.